merged_schema/HPXMLMerged.xsd

```diff
@@ -4895,6 +4895,72 @@
 					<xs:attribute name="dataSource" type="DataSource"/>
 				</xs:complexType>
 			</xs:element>
+			<xs:element minOccurs="0" name="MeasureType">
+				<xs:complexType>
+					<xs:choice>
+						<xs:element name="InstallReplace">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element ref="extension" minOccurs="0"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="AtticEncapsulation">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctsMovedToConditionedSpace" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
+							</xs:complexType>
+						</xs:element>
+						<xs:element name="FoundationEncapsulation">
+							<xs:complexType>
+								<xs:sequence>
+									<xs:element minOccurs="0" name="DuctsMovedToConditionedSpace" type="HPXMLBoolean"/>
+									<xs:element minOccurs="0" ref="extension"/>
+								</xs:sequence>
```

Comment:
FoundationEncapsulation measures move the thermal boundary of the building from the floor above the foundation space to the slab and foundation walls, converting the foundation from unconditioned to conditioned.